src/linear.js

```diff
@@ -19,38 +19,36 @@ export function linearish(scale) {
   scale.nice = function(count) {
     if (count == null) count = 10;
 
-    var d = domain(),
-        i0 = 0,
-        i1 = d.length - 1,
-        start = d[i0],
-        stop = d[i1],
-        step;
+    var d = domain();
```

Comment:
Suggestion: use existing style with single `var`.

Reply:
I'm changing all of them in https://github.com/d3/d3-scale/pull/212 anyway.